### ISSUE TITLE
Item set equipped Trigger 1 aura_env.states[1].show returning true 

### ISSUE THREAD

**Issue by oriwitt** [WeakAuras/WeakAuras2#4560]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

Trigger 1 aura_env.states[1].show returning true when the condition in Trigger 1 (4 set Triumphant Velen's Regalia) is not met. I expect output to equal false however output is true

### WeakAuras Version

Weakauras 5.6.0

### World of Warcraft Flavor

Wrath of the Lich King Classic

### World of Warcraft Region

US/NA

### Tested with only WeakAuras

- [X] Yes

### Lua Error

```Text
No error, just incorrect output
```

### Reproduction Steps

New WA Display Text %c
Update Custom text on Trigger update
Custom Function 
function ()
print(aura_env.states[1].show)
end

Trigger
Trigger 1
Type Item > Item Set Equipped
Checked Item Set ID 142
Checked >= 4

### Last Good Version

NA

### Export String

```Text
!WA:2!1rv3QnUru414lArTqt8fb6plysPlBOqOBinbk0crUsTzXRDQKsZwkf5rAoYE2knZ0zg5eVxkAH968i4R7v(ripbcZ(eKhH9jONrYP)XgDZCoN5mNZ38D(g1XVxrpAp6R2tjkn5moOgFE0Wth5j)CktlZjlIGRmXzcvbXelJnSciMUGtkyPXMzkqptKtxSmTuBef2CLhMvYtnmb)X750h)Kkg38ysPIed857RnedO)5N8l7Jh9Y9Cao1XXzTEgHkU8NgNLPbZN8GveE6mH6mbE2KbEJI8cK9VFe161rU1)GJZLuSrbWCGBULuIrjgw6fmQzM7jORI0as9rbiIugNKmgNPN54IlgNkJInDkO0JEKAJ5R2UIzGIrKcOJC7snepfq(ILgI3JX8jR)pU3QBwgiq6rCjpazmq7i3g(TsMuc0yHeueeg7(nFD1Db7D4skKuMLfTa3979gEM)5dDnOJRT13QltAUpHykSRwfp4KWO4WOtcIUXIN7kZKgV7a7KRTwHG5u6bp5WdwF3EnrMuPLqE(PuTZFx9Zuaw9GWZ8goS(9QofZTpMSBjsmj2baOUUKVHvCwHtLx0mXNdhq4lKBtASJA3)zck8NpqU79p7ke0yKjGjYhD)jLdtjPlIZYfc1SR1qEwR0WDCu04N5Mj4g5w(k2l7)dLekwos)Oi5h)wky6gdxUGdU5cc1pXqYX79obfL5gMJlsjP7u)vobP5eT2A5QzVeSgv2ofIoF4gf7ZBvSDxJsmuWow2iQCwQGPOHDo6AWUFnHZScqb)O6pW)gGOHqJc4tnZEN6hwF8Ak((WUDSDAR0qQGt1v28S1O(y3ccJ7x)Tnzx7v7x)D46w))iVbjfqXj5)iweSC)XsBHyTGs(P3pblvqkZEIUQRAVsDsWJMXM68wPr5gJa7PPQxGV7yzlCh65hT0isN32(3D(n7YOR2OfW39g1InvFvMcfNijGZQUVP95UVnK9(MeoiWZBu1Lcf9cfrwDXgJdkXYb8N6p(WGJZcZx9VG2UFwAqQixOEAx8Bv78zqtGoD60DjkQbAZpa(93FfJ3IFeKolrLFqZ8s)r7u)W1OBcj9xNI)tKt7Pd(Y9pA)VO38x)8)6d
```

**Comment by InfusOnWoW** [comment]
We don't have anyone playing Classic atm, though I've tested the aura and the trigger seems to work as exptected. I do not know why you care about aura_env.states[1].show's content. But obviously that print will only be executed when the trigger is active and thus can only ever print true.

**Comment by oriwitt** [comment]
I'm using it as a conditional for logic in the actual WA, the sample posted was the most distilled example of this 'bug' I could find. In Display > Custom Function there is code for other trigger types that work as (I think?) is intended ie. local crit_damage = 1+((1.5) * (aura_env.states[7].show and 1.03 or 1) - 1) * (aura_env.states[6].show and 2 or 1) give I think a correct true value when true and false or nil when false. Am I not correctly understanding the intended function of the item set equipped trigger?

**Comment by InfusOnWoW** [comment]
Well it wasn't a good example. If you are asking how things work, you are wrong to do so in a bug tracker.

**Comment by oriwitt** [comment]
Excuse me for being ignorant. Afaik it was a bug, the issue was raised in discord and Asakawa suggested a lodge a ticket. As it turns out the issue was a negative item ID being used in the item ID input which I had gleaned from the suggested wowhead.com/wotlk/item-sets page. As I was interested only in the 245 or 258 versions I neglected to consider the correct item set ID to use was 849 from the ilevel 232 set. Perhaps I can suggest the instructions or documentation reflect these steps required more clearly. Thank you but the issue is now resolved
